src/components/ExceededCommentLength.js

```diff
@@ -1,39 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import _ from 'underscore';
 import CONST from '../CONST';
 import Text from './Text';
 import styles from '../styles/styles';
-import stylePropTypes from '../styles/stylePropTypes';
 
 const propTypes = {
     /** The current length of the comment */
     commentLength: PropTypes.number.isRequired,
-
-    /** Additional style props */
-    style: stylePropTypes,
-};
-
-const defaultProps = {
-    style: [],
 };
 
 const ExceededCommentLength = (props) => {
     if (props.commentLength <= CONST.MAX_COMMENT_LENGTH) {
         return null;
     }
 
-    const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
-
     return (
-        <Text style={[styles.textMicro, styles.textDanger, ...additionalStyles]}>
+        <Text style={[styles.textMicro, styles.textDanger, styles.chatItemComposeSecondaryRow]}>
```

Comment:
So basically for this and the other question you had, the reason is that I was reverting this code to previous version and you can see on [this diff](https://github.com/Expensify/App/pull/9095/files#diff-7fa482b2895f4e36a2a3ad8233496489c6de1256a8c992524e2fc072aaba43f8L612-L614) in that particular PR that we had these styles in place. 

I havent tested if removing this style would keep it same looking, but I knew that with it, it would look as we wanted 😄

Reply:
Ok that's what I thought. We can leave it as is.